File: app/src/main/java/com/example/recyclerviewhomework/model/Gallery.java
package com.example.recyclerviewhomework.model;

import android.net.Uri;

public class Gallery {

    private Uri picture;

    public Uri getPicture() {
        return picture;
    }

    public void setPicture(Uri picture) {
        this.picture = picture;
    }

    public Gallery(Uri picture) {
        this.picture = picture;
    }
}

File: app/src/main/java/com/example/recyclerviewhomework/usecases/repository/data_source/database/entity/Gallery.kt
package com.example.recyclerviewhomework.usecases.repository.data_source.database.entity

import android.os.Parcelable
import androidx.room.Entity
import androidx.room.PrimaryKey
import kotlinx.android.parcel.Parcelize

@Parcelize
@Entity(tableName = "gallery")
data class Gallery(
        @PrimaryKey(autoGenerate = true)
        val id: Int,
        val gallery: MutableList<Picture>) : Parcelable
